docs/package.json

```diff
@@ -5,12 +5,11 @@
   "author": "MUI Team",
   "license": "MIT",
   "scripts": {
-    "build": "cross-env NODE_ENV=production NODE_OPTIONS=--max_old_space_size=4096 next build --profile",
+    "build": "rimraf docs/export && cross-env NODE_ENV=production NODE_OPTIONS=--max_old_space_size=4096 next build --profile && pnpm build-sw",
     "build:clean": "rimraf .next && pnpm build",
     "build-sw": "node ./scripts/buildServiceWorker.js",
     "dev": "next dev",
     "deploy": "git push -f material-ui-docs master:latest",
-    "export": "rimraf docs/export && next export --threads=3 -o export && pnpm build-sw",
```

Comment:
@oliviertassinari Any idea what caused us to add this `--threads=3`? It doesn't seem to translate anywhere in the `output: 'export' setup`